proposals/compilation-hints/Overview.md

```diff
@@ -49,6 +49,22 @@ It is expected and even desired that not all functions are annotated to keep thi
 *Note: This should be moved to `metadata.function.compilation_order` without the byte offset if such a namespace will be supported by custom annotations.*
 
 
+#### Text format
+
+Instead of a binary string representation, these hints can also be provided using a more human readable notation in text format:
+```
+@metadata.code.compilation_order(
+  (priority 1)
+  (hotness 100)
+)
```

Comment:
Looks like the current branch hints have this form:
```wat
(@metadata.code.branch_hint "\01")
(..instruction attached to..)
```
Perhaps we can follow that? That is,
```suggestion
(@metadata.code.compilation_order
  (priority 1)
  (hotness 100)
)
```

Reply:
Oh, of course. I misremembered the syntax.